scripts/manhole-cli

```diff
@@ -27,7 +27,7 @@ for sig, num in vars(signal).items():
         SIG_NUMBERS.add(num)
 
 
-def parse_pid(value, regex=re.compile(r'^(/tmp/manhole-)?(?P<pid>\d+)$')):
+def parse_pid(value, regex=re.compile(r'^(/tmp(/.*?)?/manhole-)?(?P<pid>\d+)$')):
```

Comment:
indeed it seems the path can be just free-form, also since `Manhole.install` supports arbitrary paths